src/prommis/uky/uky_flowsheet.py

```diff
@@ -2614,11 +2610,12 @@ def add_costing(flowsheet):
 
     # Leaching costs
     # 4.2 is UKy Leaching - Polyethylene Tanks
+    tank_volume = value(
+        units.convert(flowsheet.fs.leach.volume[0, 1], to_units=units.gal)
+    )
     L_pe_tanks_accounts = ["4.2"]
     m.fs.L_pe_tanks = UnitModelBlock()
-    m.fs.L_pe_tanks.capacity = Var(
-        initialize=value(flowsheet.fs.leach.volume[0, 2]), units=units.gal
-    )
+    m.fs.L_pe_tanks.capacity = Var(initialize=tank_volume, units=units.gal)
```

Comment:
Why are you using a constant value here, and not passing the leach tank volume directly? This means that the costing is static and will not be updated if you try to optimize the model for cost.